src/MicrosoftTeams.ts

```diff
@@ -822,6 +890,17 @@ namespace microsoftTeams {
             failureCallback?: (reason?: string) => void;
         }
 
+        export interface AuthenticationResultParameters {
+            /* reason Specifies a reason for the authentication failure.If specified, the frame that initiated the authentication pop- up receives this value in its callback*/
```

Comment:
NIT: Please fix typos and use standard /** */ comment style as described here:
https://github.com/airbnb/javascript#comments

Reply:
Same for other comments below.